Kernel/Syscalls/sigaction.cpp

```diff
@@ -325,4 +325,25 @@ ErrorOr<FlatPtr> Process::sys$sigtimedwait(Userspace<sigset_t const*> set, Users
     return info_value.si_signo;
 }
 
+ErrorOr<FlatPtr> Process::sys$sigsuspend(Userspace<sigset_t const*> mask)
```

Comment:
The commit message mentions: 

> but as `sigsuspend` features several (currently unimplemented) restrictions on how returns work,

We should document the unimplemented restrictions here, or somewhere?